### PR TITLE
Rehaul/admin preferences refresh

### PR DESCRIPTION
**Pull request by ikusteu**
Includes #659 
Includes #657 
Applies the UI package components and Tailwind styles to the `admin_preferences` page, with some minor fix-ups along the way. This is done to accommodate the country codes dropdown component, but also as a refresher of the entire section (to remove the MUI once and for all)

### CODE REVIEW

File: packages/client/src/pages/admin_preferences/index.tsx [hunk 1]
```diff
@@ -106,176 +71,124 @@ const OrganizationSettings: React.FC = () => {
...
-  },
-  closeButton: {
-    transform: "translate(-150%,-25%)",
-    marginTop: theme.spacing(2),
-    marginBottom: theme.spacing(1),
-    width: "0px",
+// #region fieldSetup
+const generalFields: FormSectionFieldProps[] = [
+  {
+    name: "dislpayName",
```
Comment (silviot) [inline packages/client/src/pages/admin_preferences/index.tsx:151]: Typo

File: packages/client/src/components/atoms/FormSection/FormSection.stories.tsx [hunk 2]
```diff
@@ -2,36 +2,70 @@ import React from "react";
...
-import { OrganizationLabel } from "@eisbuk/translations";
 
 export default {
   title: "Form Section",
   component: FormSection,
 } as ComponentMeta<typeof FormSection>;
 
+const generalFields = [
+  {
+    name: "dislpayName",
```
Comment (silviot) [inline packages/client/src/components/atoms/FormSection/FormSection.stories.tsx:17]: typo

**Review by silviot — CHANGES_REQUESTED** [review]
Looks great! (except for the typo)

As a side note, it' soooo painful to type in these textareas rendered by React. It's a show stopper. It's my personal main reason a switch to Svelte is required.

**Review by fadwamahmoud — APPROVED** [review]

**Review by delasous — APPROVED** [review]
LGTM

**Review by silviot — APPROVED** [review]

**Pull request merged**